modules/NuGetPackageVerifier/console/Program.cs

```diff
@@ -174,13 +182,17 @@ public static int Main(string[] args)
                             var package = packagePair.Key;
                             logger.LogInfo("Analyzing {0} ({1})", package.Id, package.Version);
 
+                            PackageSignRequest signRequest = null;
+                            signRequestManifest?.PackageSignRequests.TryGetValue(packagePair.Value.FullName, out signRequest);
```

Comment:
`out var signRequest`?

Reply:
Would be a compiler error because I'm using the elvis operator.